internal/cli/atlas/atlas_darwin.go

```diff
@@ -16,4 +16,4 @@
 
 package atlas
 
-const deprecatedMessage = "\u26A0 There's a new, dedicated Atlas CLI available for Atlas users.  Install the Atlas CLI to enjoy the same capabilities and keep getting new features. Run \"brew install mongodb-atlas-cli\" or visit https://dochub.mongodb.org/core/migrate-to-atlas-cli. Atlas commands for MongoCLI are now deprecated, but they will keep receiving support for 12 months (until April 30, 2023).\n\n"
+const deprecatedMessage = "\u26A0 There's a new, dedicated Atlas CLI available for Atlas users.  Install the Atlas CLI to enjoy the same capabilities and keep getting new features. Run \"brew install mongodb-atlas-cli\" or visit https://dochub.mongodb.org/core/migrate-to-atlas-cli. Atlas commands for MongoCLI are now deprecated, but you can keep using them for 12 months (until April 30, 2023).\n\n"
```

Comment:
Do we need to take out the part about the brew command since that won't be supported at launch?

Reply:
I will use the same message that we are using for linux. I will update the message once brew formula is ready